node/test/polkadot-simnet/test/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "polkadot-simnet-test"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+# substrate primitives
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
@ordian iiuc this would cause the same issues that I accidentally introduced via #2962 and you resolved in https://github.com/paritytech/polkadot/pull/3443/commits/505b46c52939c770bd89db5b0eb9bf4f931ff35a

Reply:
I don't think so, that issue was related to `trybuild`, this is a separate test

Reply:
My bad, ignore this then.